src/model_2/train.py

```
@@ -0,0 +1,70 @@
+    }
+    
+    return model, metrics
+
+if __name__ == "__main__":
+    print("--- Starting Modeling Pipeline ---")
+    data = get_processed_data()
+    X, y = transform_data(data)
+    
+    model, scores = train_baseline_model(X, y)
+    
+    print(f"Model Training Complete.")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)